Project.toml

```diff
@@ -1,11 +1,14 @@
 name = "POMDPModelTools"
 uuid = "08074719-1b2a-587c-a292-00f91cc44415"
 authors = ["JuliaPOMDP Contributors"]
-version = "0.2.0"
+version = "0.2.1"
 
 [deps]
+Compose = "a81c6b42-2e10-5240-aca2-a61377ecd94b"
```

Comment:
adding `Compose` as a dependency made the tests pass locally on my machine (but we may not want to have it there forever).

Reply:
Maybe just move them to `extras`?

Reply:
This should be fixed by JuliaPOMDP/POMDPModels.jl#57

Reply:
This should be fixed by JuliaPOMDP/POMDPModels.jl#57

Reply:
Ok, so we it should be safe to remove Compose again alltogether? @zsunberg